crates/uv-python/src/interpreter.rs

```diff
@@ -677,6 +677,14 @@ impl Display for StatusCodeError {
 pub enum Error {
     #[error("Failed to query Python interpreter")]
     Io(#[from] io::Error),
+    #[error(
+        "Broken symlink at `{}`, was the underlying Python interpreter removed?\n\n{}{} Consider recreating the environment (e.g., with `{}`)",
+        _0.user_display(),
+        "hint".bold().cyan(),
+        ":".bold(),
+        "uv venv".green(),
+    )]
+    BrokenVenvSymlink(PathBuf),
```

Comment:
Doesn't specializing this case mean we're not improving the error message when it's not a virtual environment? I'm curious why you chose this instead of having conditional display of the hint? Either with `BrokenSymlink(PathBuf, bool)` or by deferring that `pyvenv.cfg` lookup to error display time.

Reply:
I've only seen the error show in uv for venvs, but it doesn't hurt to cover the non-venv case, too.

In the test the error looks suboptimal too (we're ignoring the explicitly passed python interpreter is unexpected, I checked both `uv venv` and `uv pip compile`), but that's a different problem (and the behavior for this case does not change).